### PR TITLE
TINY-8371: Swap SVG logo to a file

### PR DESCRIPTION
**Pull request by TheSpyder**
Related Ticket: TINY-8371 (#7573)

Description of Changes:
* Enable file loader for both webpack and rollup
* Extract the SVG to a file
* Swap the now-complex HTML string to an Alloy spec
* Adjusted the branding CSS to fix an issue where the top of the logo was cut off because the image was too close to the top of the status bar

Before:
before
After:
after

Pre-checks:

* [x] ~Changelog entry added~
* [x] Tests have been added (if applicable)
* [x] ~Branch prefixed with `feature/` for new features (if applicable)~

Review:
* [ ] Milestone set
* [ ] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts
Comment (lnewson) [inline modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:12]: Should this be a relative path since it's something within the silver theme? Normally anything within the `main` dir would need to be relative, but our regular codestyle doesn't really cover this case, so just thought it was worth asking.

File: modules/tinymce/tools/modules/grunt-webpack.js
Comment (lnewson) [inline modules/tinymce/tools/modules/grunt-webpack.js:33]:
```suggestion
          test: /\.(svg)$/i,
```
We definitely don't want regular images using the raw-loader, so just dropping it back to what we need for now.

File: patches/@ephox+bedrock-server+11.5.0.patch
Comment (lnewson) [inline patches/@ephox+bedrock-server+11.5.0.patch:10]: For others reading, this was needed as Bedrock can't currently load images and for TinyMCE we're stuck on an older version of Bedrock atm so we can't just add this to bedrock and upgrade. So this was a solution Andy and I discussed that we thought was best until we can upgrade, as this shouldn't be needed elsewhere for older bedrock versions.

**Review by lnewson — APPROVED** [review]

File: modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts
Comment (TheSpyder) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:12]: The problem is that when looking outside of `main/ts` the relative path might change when this source is compiled into the `lib` folder. I figured a named path like this was more consistently reliable.

Comment (lnewson) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:12]: Yeah that's a good point, lets leave it as is then.

File: modules/tinymce/tools/modules/grunt-webpack.js
Comment (TheSpyder) [reply modules/tinymce/tools/modules/grunt-webpack.js:33]: ah yeah this was added before the swap to `raw`, thanks

**Review by spocke — APPROVED** [review]

File: patches/@ephox+bedrock-server+11.5.0.patch
Comment (spocke) [reply patches/@ephox+bedrock-server+11.5.0.patch:10]: Didn't know that we had all those types of imports already in bedrock.

File: modules/oxide/src/less/theme/components/statusbar/statusbar.less
Comment (lnewson) [inline modules/oxide/src/less/theme/components/statusbar/statusbar.less:84]: One thing I just thought of... since this will be shared between 5.x and 6.x skins, this might need to be a variable and adjusted accordingly. I don't know enough about how that's all setup though, but just flagging it as a possible issue.

Comment (lnewson) [reply modules/oxide/src/less/theme/components/statusbar/statusbar.less:84]: Actually, that's only an issue if we're showing the text for 5.x. I don't believe we are so this probably isn't an issue.

Comment (TheSpyder) [reply modules/oxide/src/less/theme/components/statusbar/statusbar.less:84]: I think the 6.x skin PR can worry about that after the logo has merged :)

**Comment by TheSpyder** [comment]
@noxuhax @lostkeys  I will wait for design review before merging this

**Review by ltrouton — APPROVED** [review]

File: modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts
Comment (noxuhax) [inline modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:36]: Is there a particular reason for making this link unfocusable via keyboard? I might be wrong but it seems that combining both `tabindex=-1` and `aria-label` attributes is meaningless since screen reader won’t be able to read the lable without the link getting focused in the first place

**Review by noxuhax — APPROVED** [review]
From the visual point of view all seems good — great job on catching a pixel being cut off from the top of the logo. Logo size will be adjusted in the legacy `tinymce-5` skin to better fit into the smaller status bar.

File: modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts
Comment (noxuhax) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:36]: Oh, and if we decide to remove `tabindex` attribute it would be a good idea to add `role="presentation"` to the svg as @spocke suggested previously.

Comment (lnewson) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:36]: The tabindex=-1 is required because it's handled via our UI framework (Alloy) not via native keyboard navigation. That said though, now this is an Alloy spec this isn't quite the correct way to do this. Instead, we should be adding the `Focusing` behaviour instead.

Comment (TheSpyder) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:36]: I didn't add or remove any attributes, I just converted the existing attributes to Alloy spec:
https://github.com/tinymce/tinymce/blob/8c13482153500dba4a1b16c033c190cc72b91417/modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts#L23
This discussion makes me happy, as does the new commit to change the URL. These problems existed before but were not noticed.

Comment (lnewson) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts:36]: Sorry I had noticed that as well and should have noted they were there beforehand in my reply as well 😞

Comment (lnewson) [reply modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts]: Was talking to @TheSpyder about this via slack. I've gone ahead and swapped it to use the `Focusing` behaviour as I mentioned.

As for the `role="presentation"` part, if that was an a11y issue then it'd also be an issue for all our toolbar icons, etc... as well so we should probably investigate that and follow up in a separate Jira instead. Does anyone have any objections to following that up via a separate Jira?

Now lastly, and this isn't related to these changes but more the original change, but the is no focus style for the logo so you can't see what's focused. That does seem like an a11y issue so we probably need to chase that up with the design team and see if some styles should be added for that. We should handle that after this PR is merged into the existing PR though.

**Pull request merged**
